src/components/AppsMenu/index.jsx

```diff
@@ -35,15 +36,16 @@ const AppsMenu = () => {
         <Menu
           open={isOpen}
           anchorEl={buttonRef.current}
-          container={buttonRef.current}
+          container={containerRef.current}
           getContentAnchorEl={null}
           onClose={toggleMenu}
           anchorOrigin={{
             vertical: 'bottom',
             horizontal: 'left'
           }}
           transformOrigin={{
-            vertical: -10
+            vertical: -10,
```

Comment:
nit: still strange this `-10`, mayb we should change the default style somewhere 🤔

Reply:
It is because we want to menu not to be sticked to the button.